### ISSUE TITLE
How to train in a single GPU?

### ISSUE THREAD

**Issue by shipeng-liu** [charlesCXK/TorchSSC#4]
Hello, I am wondering if I want to train the model in a single gpu computer, should I config anything except the NGPUS?

**Comment by charlesCXK** [comment]
Hi, you may need to change the batch size and the learning rate in the file `model/sketch.nyu/config.py`. 
For example, 
```python
C.batch_size = 4 ==> C.batch_size = 2
C.lr = 0.1 ==> C.lr = 0.05
```
Or you could set other batch sizes according to your available GPU memory.

**Comment by shipeng-liu** [comment]
Thank you
